io.github.jean28518.Linux-Assistant.yml

```diff
@@ -0,0 +1,37 @@
+app-id: io.github.jean28518.Linux-Assistant
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+
+command: linux-assistant
```

Comment:
What's the purpose of installing `linux-assistant.sh` then? Or should this be that instead?